### ISSUE TITLE
Replace from, to parameters in mongo driver with from, window form

### ISSUE THREAD

**Issue by jmrunge** [inaes-tic/mbc-mosto#42]
Now, functions from mongo driver receive boundaries in the from(date) and to(date) form.  They will need to change to from(date) to window(hours).

Ie: from 10:00:15, window +4hs

**Comment by jmrunge** [comment]
Is this done? If so, please mention PR and close issue.  Thanks.

**Comment by Lacrymology** [comment]
Now it works like this:

```
start(+4hs)
getWindow({ [from: date,] [to: date,] [span: hs] })
```

**Comment by Lacrymology** [comment]
Fixed in #62
